File: Backend/Larpex.Shared/ModelDto/StripeRequestDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Larpex.Shared.ModelDto
{
    public class StripeRequestDto
    {
        public string? StripeSessionUrl { get; set; }
        public string? StripeSessionId { get; set;}
        public string ApprovedUrl { get; set; }
        public string CancelUrl { get; set; }
    }
}

File: Frontend/src/entities/StripeRequestDto.ts
interface StripeRequestDto {
  stripeSessionUrl: string;
  stripeSessionId: string;
  approvedUrl: string;
  cancelUrl: string;
  orderDto: {
    orderId: string;
    orderAmount: number;
    eventId: number;
  };
}

export default StripeRequestDto;
